recipes-bsp/imx-atf/imx-atf_2.4.bb

```diff
@@ -48,6 +49,9 @@ EXTRA_OEMAKE += 'LD="${@remove_options_tail(d.getVar('LD'))}"'
 
 EXTRA_OEMAKE += 'CC="${@remove_options_tail(d.getVar('CC'))}"'
 
+# Set the UART to use during the boot.
+EXTRA_OEMAKE += 'IMX_BOOT_UART_BASE=${ATF_BOOT_UART_BASE}'
```

Comment:
If someone sees `IMX_BOOT_UART_BASE=INVALID` in the build, they might think something is wrong and waste time figuring out that it is not wrong. Can you change this so nothing gets added to `EXTRA_OEMAKE` in this case?

Reply:
I don't want to let this be "skipped" behind the scenes.

We have values for the supported SoCs inside `imx-base.inc`, and if a new SoC is added, it will fail to build as a platform does.

Reply:
Currently `IMX_BOOT_UART_BASE=INVALID` is passed for everything but `mx8m-generic-bsp`. Are you saying that you are not finished and you plan to define the rest?

Reply:
No. Those are the only ones I could find. See:
```
otavio@yocto-env ..x-atf/2.6+gitAUTOINC+c6a19b1a35-r0/git (git)-[lf_v2.6] % git grep '^IMX_BOOT_UART_BASE'
plat/imx/imx8m/imx8mm/platform.mk:IMX_BOOT_UART_BASE    ?=    0x30890000
plat/imx/imx8m/imx8mn/platform.mk:IMX_BOOT_UART_BASE    ?=    0x30890000
plat/imx/imx8m/imx8mp/platform.mk:IMX_BOOT_UART_BASE    ?=    0x30890000
```
I found a wrong operator which caused an error. Could you check it?

Reply:
Okay, I think it makes sense to me now.